src/Tablebot/Plugins/Cats.hs

```diff
@@ -47,9 +51,21 @@ cat =
 
 -- | @getCatAPI@ is a helper function that turns gets a JSON object that may
 -- contain a cat image. Uses https://docs.thecatapi.com/ for cats.
+-- CATAPI_TOKEN must be provided in .env so that there are no cat limits.
 getCatAPI :: IO (Either String CatAPI)
 getCatAPI = do
-  req <- parseRequest "https://api.thecatapi.com/v1/images/search"
+  initReq <- parseRequest "https://api.thecatapi.com/v1/images/search"
+  loadEnv
```

Comment:
`loadEnv` call is unnecessary here from my testing - I think the fact that it's called at bot launch is enough